cylc/flow/config_schema.py

```diff
@@ -499,7 +505,7 @@ def get_config(output_fname, tvars, suite_fpath=None, user=True, site=True):
         A CylcConfig object.
     """
     _HOME = pathlib.Path.home() or get_user_home()
-    SITE_CONF_DIR = pathlib.Path(_HOME, "mock_cylc_global")
+    SITE_CONF_DIR = pathlib.Path(os.sep, 'etc', 'cylc', 'flow', CYLC_VERSION)
```

Comment:
Don't need to use `os.sep`, pathlib can handle `/etc`.

(infact pathlib can handle `"/etc/cylc/flow/", CYLC_VERSION`)